llvm/lib/CodeGen/CommandFlags.cpp

```diff
@@ -744,7 +744,7 @@ void codegen::setFunctionAttributes(StringRef CPU, StringRef Features,
 Expected<std::unique_ptr<TargetMachine>>
 codegen::createTargetMachineForTriple(StringRef TargetTriple,
                                       CodeGenOptLevel OptLevel) {
-  Triple TheTriple(TargetTriple);
+  Triple TheTriple(TargetTriple.str());
```

Comment:
This looks redundant - doesn't Triple's constructor do the conversion to `std::string` from Twine internally?

Reply:
> This looks redundant - doesn't Triple's constructor do the conversion to `std::string` from Twine internally?

Thanks for flagging my incomplete restoration. Fixed.